src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
```

Comment:
```suggestion
  constructor(name) {
```
it's not necessary to take health as constructor parameter